python/server/auto_generated/qgsconfigcache.sip.in

```diff
@@ -66,6 +66,13 @@ value is ``False``).
 Returns the name of the current strategy
 
 .. versionadded:: 3.26
+%End
+
+    QMap<QString, QDateTime> projects() const;
+%Docstring
+Returns projects currently in cache.
```

Comment:
Can we be more precise _what_ we will get?
Intuitively I was assuming a list of project objects from given `projects()` but your example in the PR made it clearer. I think the API docs should be precise on what we can expect from the string and the date.